tests/conftest.py

```diff
@@ -1,81 +0,0 @@
-# pylint: disable=missing-function-docstring, missing-module-docstring
-import logging
-import os
-import shutil
-import pytest
-from mpi4py import MPI
-from pyccel.commands.pyccel_clean import pyccel_clean
```

Comment:
Why is this file removed?

Reply:
I encountered confusion between my changes and mouad's changes on GitHub, so I decided to delete some files and retrieve them again from to fix that

Reply:
That's what I thought. If you do that you should take care to check the diff tab on the PR to see if there is anything showing up that you don't remember purposefully changing

Reply:
@smazouz42 This change is the reason why your tests are failing

Reply:
thank you